### PR TITLE
Relay Authorities Test

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

**Comment by AurevoirXavier** [comment]
Don't merge, I have got a patch for #385

**Comment by AurevoirXavier** [comment]
Waiting #392

File: frame/bridge/relay-authorities/src/lib.rs [hunk 1]
```diff
@@ -231,19 +236,24 @@ decl_module! {
...
-					ensure!(
-						stake >
-							candidates
-								.iter()
-								.map(|candidate| candidate.stake)
-								.max()
-								.unwrap_or(0.into()),
-						<Error<T, I>>::StakeIns
-					);
+					let mut minimum_stake = candidates[0].stake;
```
Comment (HackFisher) [inline frame/bridge/relay-authorities/src/lib.rs:241]: This is to kick out the candidate with minimum stake?

Comment (AurevoirXavier) [reply frame/bridge/relay-authorities/src/lib.rs:241]: Yes. There's a max candidate count.

File: bin/node/cli/src/command.rs [hunk 2]
```diff
@@ -175,9 +155,6 @@ pub fn run() -> sc_cli::Result<()> {
 		}
 		Some(Subcommand::Revert(cmd)) => {
 			let runner = cli.create_runner(cmd)?;
-			let chain_spec = &runner.config().chain_spec;
-
-			set_default_ss58_version(chain_spec);
```
Comment (AsceticBear) [inline bin/node/cli/src/command.rs:180]: Why made these changes?

Comment (AurevoirXavier) [reply bin/node/cli/src/command.rs:180]: pangolin is 42

**Review by HackFisher — APPROVED** [review]

**Pull request merged**